### DISCUSSION TITLE
cutlass.nvim not overriding keys with lazyvim

### DISCUSSION THREAD

**Discussion by tommyalatalo** [LazyVim/LazyVim#489] [Q&A] [answered]
I've been using vim-cutlass for some time, which unfortunately seems to have stopped working with some recent updates of neovim. I now saw that there is a more well maintained project for neovim; [gbprod/cutlass.nvim](https://github.com/gbprod/cutlass.nvim), however when setting it up I can't get the plugin to override the keybindings (the main use I have for this is to map `m` to "cut" and `d` to "delete, so for instance  `mm` would cut a line to the clipboard and `dd`  just deletes the line to a blackhole register.

But like I said, the keymaps aren't being set correctly and `d` still copies text to the clipboard and there is this mention in the readme for cutlass:

```
It overrides the following keys to always use the black hole register: c, C, s, S, d, D, x, X.

Note that if you have already mapped these keys to something else (like we do below with x) then it will not change it again.
```

So my question is, are these keys already overridden/reconfigured by lazyvim somewhere, since my config below isn't resulting in the behavior that's expected? My current config is this:

```
return {
	"gbprod/cutlass.nvim", -- add 'cut' operation to vim
	keys = {
		{ "m", "d" },
		{ "mm", "dd" },
		{ "M", "D" },
		{ "gm", "m" },
		{ "m", "d", mode = "x" },
	},
	opts = {
		cut_key = "m",
		override_del = true,
		registers = {
			select = "_",
			delete = "_",
			change = "_",
		},
	},
}
```

**Comment by folke** [comment, marked as answer]
Those keys are not overridden by LazyVim. You probably shouldn't set those `keys=` though for this plugin. And it should not be lazy-loaded I think

**Reply by tommyalatalo** [reply]
Ah yes, setting `lazy = false` and removing the keys seems to have done the trick

Thanks :)
